docs/workload.rst

```diff
@@ -0,0 +1,193 @@
+VM Workloads Search
+==============
+
+Visualize the inventory of vSphere workloads that do not have Carbon Black Cloud sensors installed.
+
+Fetch Compute Resource by ID
+----------------------------
+
+Using a query of the ``ComputeResource`` object, you can get the compute resource by ID from your organization.
+
+::
+
+    >>> from cbc_sdk import CBCloudAPI
+    >>> from cbc_sdk.workload import ComputeResource
+
+    >>> # This is an example id that we want to query
+    >>> id = 15054425
```

Comment:
I'm not a fan of using a GET by id as an initial example given that it's not clear where to get the id value from. Can we either indicate where to fetch this value programmatically or push this example after a search of the compute resource?